resources/helm/overlays/istio-control/istio-discovery/templates/reader-serviceaccount.yaml

```diff
@@ -0,0 +1,17 @@
+# Source: istiod-remote/templates/reader-serviceaccount
+{{- if .Values.global.clusterWide }}
```

Comment:
This service account should be applied only when cluster-wide and moulti-cluster modes are enabled.

Reply:
You also forgot about closing this if statement.